extensions/iceberg/src/main/java/io/deephaven/iceberg/util/IcebergCatalogAdapter.java

```diff
@@ -172,6 +172,18 @@ public List<Namespace> listNamespaces(@NotNull final Namespace namespace) {
                 "%s does not implement org.apache.iceberg.catalog.SupportsNamespaces", catalog.getClass().getName()));
     }
 
+    /**
+     * List all {@link Namespace namespaces} in a given namespace. This method is only supported if the catalog
+     * implements {@link SupportsNamespaces} for namespace discovery. See
+     * {@link SupportsNamespaces#listNamespaces(Namespace)}.
+     *
+     * @param namespace The namespace(s) to list namespaces in.
+     * @return A list of all namespaces in the given namespace.
+     */
+    public List<Namespace> listNamespaces(@NotNull final String... namespace) {
```

Comment:
I'm not sure we need list methods that are String-y. That is, if someone is working with this programmatically, they probably can make the Namespace object.